### DISCUSSION TITLE
Is there a way to check if iOS App Privacy is completed from fastlane?

### DISCUSSION THREAD

**Discussion by sonjz** [fastlane/fastlane#17780] [Q&A] [answered]
Dropped question here too:
https://github.com/fastlane/fastlane/pull/17592#issuecomment-743299621

Got new app privacy automation in, but still have many teams that don't provide Admin access, need a way to poll them if they have completed it.
> i have it working for non-Admin 🎉
> ... but is there a way to check if App Privacy is completed?
> 
> we currently flag it in our system if we completed the app privacy for a non-Admin,
> but for other team Admin/Account Holders that have to complete this manually, i'm not quite sure how to track it.
> 
> the only place i've seen an error during deliver, which is a bit further along the process than we'd like.

**Comment by dinsen** [comment, marked as answer]
You could use this when it gets released? https://github.com/fastlane/fastlane/pull/17746

**Comment by sonjz** [comment]
🤦🏻‍♂️
thanks, didn't look there!
